<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="1">

  <testsuite name="profile"
    errors="0" failures="1" hostname="" id="" package="profile" skipped="0"
    tests="1" time="78.94" timestamp="Thu, 14 Apr 2016 11:56:11 GMT">
  
    <testcase name="redirect you to your user profile logged clicking the button" time="78.94" assertions="2">

      <failure message="Timed out while waiting for element &lt;fieldset.auth-form-email &gt; div.form-group &gt; input#loginFormEmail&gt; to be present for 60000 milliseconds.">    at Object.module.exports.redirect you to your user profile logged clicking the button (C:\Users\Emilio\nightwhatch\src\tests\profile.js:11:3)
    at Module.call (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\module.js:63:34)
    at C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:796:13)
    at C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\Users\Emilio\AppData\Roaming\npm\node_modules\nightwatch\node_modules\q\q.js:125:13)
    at nextTickCallbackWith0Args (node.js:420:9)</failure>

    
    </testcase>
  
  
  </testsuite>
</testsuites>
